<?xml version="1.0"?>
<launch>

    <arg name="output" default="screen"/>

    <node pkg="scan_line_detector" type="scan_line_detector_node" 
          name="scan_line_detector_node" output="$(arg output)">

        <!-- laserscan -->
        <param name="scan_topic" value="/scan"/>
        <param name="laser_min_range" value="-1"/>
        <param name="laser_max_range" value="-1"/>
        <param name="voxel_size" value="0.001"/>
        <param name="max_laser_points" value="10000"/>
        <param name="robot_frame_id" value="/base_footprint"/>

        <!-- map -->
        <param name="resolution" value="0.025"/> 

    </node>

</launch>